orchestration/values.yaml

```diff
@@ -0,0 +1,11 @@
+cron:
+  enable: true
+  # Every Sunday at 3:30am
+  schedule: '30 3 * * Sun'
+  timezone: 'America/New_York'
+dataflow:
+  region: us-west1
+  workerMachineType: n1-standard-1
+  autoscaling:
+    minWorkers: 4
+    maxWorkers: 8
```

Comment:
I based these values off of what clinvar uses. Let me know if there's anything I should change!

Reply:
Makes sense to me as a first pass